data/system-metastasis-codes-json.json

```diff
@@ -0,0 +1,12 @@
+{
+    "SNOMED": {
```

Comment:
For the UTSW records, this should work but typically each value has many codes associated with it and I would expect in future we would expand these codes to be more comprehensive. In this case we would have to change the structure of this, @ChristineDuong @zlister , should we leave it as is for now or update it now with the expectation that there will be more codes eventually?

Reply:
If there are more codes associated with a value, the structure shouldn't need to change. For example, if SNOMED has two codes for `bone`, you can just add that key like so:

```
{
    "SNOMED": {
        "94222008": "bone",
        "94381009": "bone"
    }
}
```

Reply:
Sorry about that! Sounds good